apps/web/src/pages/invites/v2/MembersInvitePage.tsx

```diff
@@ -13,7 +13,11 @@ import { CopyInviteLink } from './CopyInviteLink';
 import { MemberInviteForm } from './MemberInviteForm';
 import { buildInviteHref, generateInviteLink } from './MembersInvitePage.utils';
 
-export function MembersInvitePage() {
+interface IMembersInviteProps {
+  shouldHideTitle?: boolean;
+}
+
+export function MembersInvitePage({ shouldHideTitle }: IMembersInviteProps) {
```

Comment:
<!-- Notes are always non-blocking and simply highlight something the reader should take note of.  -->
🗒 **note (non-blocking):** This is not a pattern we should adopt in general, but for the sake of a minimal wrapper for it to work for the old and new use cases, this was a cheap and effective solution‏